ambry-mysql/src/main/resources/AmbryContainerStorageStats.ddl

```diff
@@ -19,43 +19,70 @@
  */
 CREATE TABLE IF NOT EXISTS AccountReports
 (
-    clusterName  VARCHAR(25) NOT NULL,
-    hostname     VARCHAR(30) NOT NULL,
-    partitionId  INT         NOT NULL,
-    accountId    INT         NOT NULL,
-    containerId  INT         NOT NULL,
-    storageUsage BIGINT      NOT NULL,
-    updatedAt    TIMESTAMP   NOT NULL,
+    clusterName          VARCHAR(25) NOT NULL,
+    hostname             VARCHAR(30) NOT NULL,
+    partitionId          INT         NOT NULL,
+    accountId            INT         NOT NULL,
+    containerId          INT         NOT NULL,
+    storageUsage         BIGINT      NOT NULL, /* logical storage usage */
+    physicalStorageUsage BIGINT      NOT NULL, /* physical storage usage */
+    numberOfBlobs        BIGINT      NOT NULL, /* number of blobs, include blobs at all states, even for compaction */
```

Comment:
Hi sorry for the comment after merging, but I just remembered this. How will you synchronize the alter table and the code deployment.

If you alter the table before the code is deployed, the old code will fail since these new columns are required.
If you alter the table after the code is deployed, the new code may fail since columns don't exist in the table yet.

Could we resolve this by placing default values?